_data/redirects.yml

```diff
@@ -0,0 +1,5 @@
+"": "en"
+cn: "zh-hans"
+jp: "ja"
+my: "ma"
```

Comment:
Redirects from `my` end up with 404. The problem is that we don't have the `ma` folder specified here but we have `ms` instead. Likely [the current folder name is correct](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=ms), so `ms` should be used here.